CHANGES.md

```diff
@@ -53,6 +53,9 @@
 - odoc rules now about `ODOC_SYNTAX` and will re-run accordingly (#6010, fixes
   #1117, @emillon)
 
+- Add `%{coq:...}` macro for accessing data about the configuration about Coq.
```

Comment:
Is the double about here needed?

Reply:
Doesn't seem to be the case in the changelog.